### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by jkrlr**
#92 
**After Making Changes Redame looks likes as shown here**:
<br>
Screenshot from 2021-03-16 09-06-26

Screenshot from 2021-03-16 09-07-00

Screenshot from 2021-03-16 09-07-53
Screenshot from 2021-03-16 09-13-12

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,20 +55,19 @@ A PythonAnywhere Account (for deploying the project on a live system)
...
    * * Download the Windows x86 executable installer for 32-bit version of Windows.
 
 
-* Make sure to check "Add Python 3.x to Path" in the setup window of the Installer.
+   * * Make sure to check "Add Python 3.7 to Path" in the setup window of the Installer.
 
 Verify the installation from the command prompt using following command :
 ```
-python --version
+python3.7 --version
```
Comment (jkrlr) [inline README.md:70]: Added link to download the python3.7 as original project has been made in python3.7. So, that existing requirements.txt file can work fine.

File: README.md [hunk 2]
```diff
@@ -78,8 +77,29 @@ And the installed version of python will be printed.
...
+
+
+<details><summary>Step 4 : Fork the Repository </summary>
+
+Click on ![Fork](https://user-images.githubusercontent.com/63921263/110382285-b07bba80-8080-11eb-8407-d354849c1753.png)
+to fork [this Repo](https://github.com/chiraag-kakar/sharenlearn)
+
+</details>
+
+
```
Comment (jkrlr) [inline README.md:97]: Added Link to download the Git  to maintain inheritance as readme contains links to download the other prerequisites.

File: README.md [hunk 3]
```diff
@@ -78,8 +77,29 @@ And the installed version of python will be printed.
...
 
+<details><summary>Step 3 : Installing Git </summary>
 
-<details><summary>Step 3 : Creating Project Directory </summary>
+Click on [Download Git](https://git-scm.com/downloads)
+
+</details>
+
+
+---
```
Comment (jkrlr) [inline README.md:87]: Added Link to download the Git to maintain inheritance as readme contains links to download the other prerequisites.

File: README.md [hunk 4]
```diff
@@ -78,8 +77,29 @@ And the installed version of python will be printed.
...
+
+
+<details><summary>Step 4 : Fork the Repository </summary>
+
+Click on ![Fork](https://user-images.githubusercontent.com/63921263/110382285-b07bba80-8080-11eb-8407-d354849c1753.png)
+to fork [this Repo](https://github.com/chiraag-kakar/sharenlearn)
+
+</details>
```
Comment (jkrlr) [inline README.md:95]: Seperate and enhance Fork part from clone and fork the repository

File: README.md [hunk 5]
```diff
@@ -98,13 +118,12 @@ cd myproject
...
  
  
 ```
-git clone https://github.com/chiraag-kakar/sharenlearn.git
+$ git clone https://github.com/<your-github-username>/sharenlearn.git
 ```
-Note: The cloned repo directory need to be renamed as "sharenlearn".
 
 
 </details>
```
Comment (jkrlr) [inline README.md:129]: Replace `chirag-kakar` to `<your-github-username>` in clone url

File: README.md [hunk 6]
```diff
@@ -113,20 +132,50 @@ Note: The cloned repo directory need to be renamed as "sharenlearn".
...
+ 
+
 ```
-python -m venv myvenv
+$ git remote add upstream https://github.com/chiraag-kakar/sharenlearn.git
 ```
-Activating "myvenv" :
+
+
+</details>
```
Comment (jkrlr) [inline README.md:158]: Add reference to main repo for sync the local master to be master of main repo.

File: README.md [hunk 7]
```diff
@@ -113,20 +132,50 @@ Note: The cloned repo directory need to be renamed as "sharenlearn".
...
+$ virtualenv myvenv -p python3.7
 ```
+Activating "myvenv" :
+* For **Windows cmd** :
+   ```
+   > myvenv\Scripts\activate
+   ```
 Command to deactivate "myvenv" :
 ```
 deactivate
```
Comment (jkrlr) [inline README.md:181]: Add command to create the virtualenv for python3.7

File: README.md [hunk 8]
```diff
@@ -194,6 +236,14 @@ python manage.py runserver
...
 
 
+---
+
+
+### 💡️ **Pro Tip!** 
+  * Always keep your master branch in sync with the main repository (by running `$ git pull upstream master` on your local master branch). 
+  * Always create a new branch before making any changes (`$ git checkout -b <new-branch-name>`), never ever make any changes directly on the master branch.
+
+
```
Comment (jkrlr) [inline README.md:246]: Add Pro Tip!  to keep up to date local master to be master of original repo.

**Review by amanjha8100 — APPROVED** [review]

**Pull request merged**